src/hooks/useLineItems.js

```diff
@@ -9,13 +9,12 @@ const useLineItems = () => {
   const {
     apiPath,
     csrf,
-    applicationState,
+    lineItems,
     setApplicationState,
   } = useContext(CheckoutContext);
-  const [isLoading, setLoading] = useState(false);
+  const [loadingLineItems, setLoadingLineItems] = useState(false);
 
-
-  const updateQuantity = useCallback(async (quantity, lineItemKey) => {
+  const updateLineItemQuantity = useCallback(async (quantity, lineItemKey) => {
```

Comment:
Not sure that this needs to be async / await as we are only fetching data and setting state in the resp.